avro-serializer/src/main/java/io/confluent/kafka/serializers/AbstractKafkaAvroDeserializer.java

```diff
@@ -37,7 +37,7 @@
 import io.confluent.kafka.schemaregistry.client.rest.exceptions.RestClientException;
 import kafka.utils.VerifiableProperties;
 
-public abstract class AbstractKafkaAvroDeserializer<W> extends AbstractKafkaAvroSerDe {
+public abstract class AbstractKafkaAvroDeserializer extends AbstractKafkaAvroSerDe {
```

Comment:
Can this stay generic, and we just parameterize it with `<Object>` from `KafkaAvroDeserializer`?

Reply:
I suppose so.  But I'd rather just remove the type param to be consistent with `AbstractKafkaAvroSerializer` since that is how it was before.   We can look at re-adding type params with 6.0 perhaps.

Reply:
Sounds good!